krod/interfaces/cli.py

```
@@ -59,30 +60,21 @@ def process_command(self, command: str) -> bool:
-            self.logger.error(f"Query error: {str(e)}", exc_info=True)
-        
-        return True
+
+        elif command.startswith('vector_store'):
+            try:
+                args = command.split()[1:]
+                # Create a standalone context and invoke the command
+                ctx = vector_store_commands.make_context('vector_store', args)
+                vector_store_commands.invoke(ctx)
+                return True
+            except click.ClickException as e:
```

Ruff (0.11.9)

Undefined name `click`

(F821)